spaces/S000209/properties/P000016.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000209
+property: P000016
+value: true
+---
+
+$X$ is homeomorphic to the one-point compactification of {S83}.
```

Comment:
Same as previously, one shouldn't use the name "one-point compactification" and especially not with "the" since its not in any way the only one-point (non-Hausdorff) compactification.